cmd/queue/main.go

```diff
@@ -155,9 +155,21 @@ func proxyHandler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, tracingEn
 
 		// Enforce queuing and concurrency limits.
 		if breaker != nil {
+			var cf func()
```

Comment:
```suggestion
			cf := func() {}
```

Then drop the ifs below?

Reply:
Done. Slightly differently